### PR TITLE
docs: improved testnet snapshot intructions

### PR DESCRIPTION
**Pull request by cosmic-vagabond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,34 @@ This guide provides instructions on how to spin up a new localnet using the Elys
...
    ```bash
    rm -rf ~/.elys && curl -o - -L https://snapshots.elys.network/elys-snapshot-main.tar.lz4 | lz4 -c -d - | tar -x -C ~/
    ```
 
+   **Devnet branch**
+
+   Use the following command to download the latest TestNet snapshot that uses the changes from the `devnet` branch:
+
+   ```bash
+   rm -rf ~/.elys && curl -o - -L https://snapshots.elys.network/elys-snapshot-devnet.tar.lz4 | lz4 -c -d - | tar -x -C ~/
```
Comment (amityadav0) [inline README.md:46]: Screenshot 2024-06-19 at 7 49 01 PM

Comment (amityadav0) [reply README.md:46]: `develop` is working fine

Comment (cosmic-vagabond) [reply README.md:46]: @amityadav0 we have to wait for changes within the `devnet` branch in the elys repo to trigger the construction of the snapshot for `devnet`

**Review by amityadav0 — APPROVED** [review]
LGTM

**Pull request merged**